src/test/java/org/folio/edge/sip2/repositories/FeeFinesRepositoryTests.java

```diff
@@ -0,0 +1,138 @@
+package org.folio.edge.sip2.repositories;
+
+import static org.folio.edge.sip2.api.support.TestUtils.getJsonFromFile;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.Mockito.any;
+import static org.mockito.Mockito.argThat;
+import static org.mockito.Mockito.when;
+
+import io.vertx.core.Future;
+import io.vertx.core.MultiMap;
+import io.vertx.core.Vertx;
+import io.vertx.core.impl.NoStackTraceThrowable;
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+import io.vertx.junit5.VertxExtension;
+import io.vertx.junit5.VertxTestContext;
+import java.util.UUID;
+import org.folio.edge.sip2.session.SessionData;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith({VertxExtension.class, MockitoExtension.class})
+public class FeeFinesRepositoryTests {
+  private static final String FIELD_TOTAL_RECORDS = "totalRecords";
+  private static final String FIELD_MANUALBLOCKS = "manualblocks";
+
+  @Test
+  public void canCreateFeeFinesRepository(
+      @Mock IResourceProvider<IRequestData> mockFolioResource) {
+    final FeeFinesRepository usersRepository =
+        new FeeFinesRepository(mockFolioResource);
+
+    assertNotNull(usersRepository);
+  }
+
+  @Test
+  public void cannotCreateFeeFinesRepositoryWhenResourceProviderIsNull() {
+    final NullPointerException thrown = assertThrows(
+        NullPointerException.class,
+        () -> new FeeFinesRepository(null));
+
+    assertEquals("Resource provider cannot be null", thrown.getMessage());
+  }
+
+  @Test
+  public void canGetManualBlocksByUserId(Vertx vertx,
+      VertxTestContext testContext,
+      @Mock IResourceProvider<IRequestData> mockFolioProvider) {
+
+    final String manualBlocksResponseJson = getJsonFromFile("json/no_manual_blocks_response.json");
+    final JsonObject manualBlocksResponse = new JsonObject(manualBlocksResponseJson);
+
+    when(mockFolioProvider.retrieveResource(any()))
+        .thenReturn(Future.succeededFuture(new FolioResource(manualBlocksResponse,
+            MultiMap.caseInsensitiveMultiMap().add("x-okapi-token", "1234"))));
+
+    final SessionData sessionData = SessionData.createSession("diku", '|', false, "IBM850");
```

Comment:
might want to use TestUtils.getMockedSessionData() if none of the arguments are specific to this test.  There are other instances like this in other tests.

Reply:
Shoot. Forgot about that. Agreed. I'll update the references.